packages/main/src/plugin/api/container-info.ts

```diff
@@ -42,7 +42,7 @@ export interface ContainerInfo {
   Names: string[];
   Image: string;
   ImageID: string;
-  Command?: string;
+  Command?: readonly string[];
```

Comment:
From my understanding, this interface is extracted from dockerode definiton

https://github.com/codershiba/DefinitelyTyped/blob/2639c808780fbe2c5ac588c14c7344fd8fdd1c1c/types/dockerode/index.d.ts#L351-L356

Reply:
AFAIK, you're totally right. However, I changed it to `readonly string[]` because `listContainers` dynamically selects among various providers, leading to the use of workarounds such as `podmanContainer.Command?.length > 0 ? podmanContainer.Command[0]: undefined`. In my opinion, `ContainerInfo` should use an array to represent a command even though `dockerode` returns a string because it enables us to not lose data(in this case other parts of the command).

Reply:
Yes I looked at the podman repository and they are indeed sending an array compared to dockerode definition.

https://github.com/containers/podman/blob/17f36a369035b4d5c27f416af9cb961ede807eed/pkg/domain/entities/types/container_ps.go#L14-L15

I would be in favour to get something closer to what podman provide, the changes you are providing only touch the internal api, so it would not have much side effect, however the `ContainerInfo` is also provided to the `extension-api`. 

https://github.com/containers/podman-desktop/blob/689b3f80eb609b3dc3fece61373f83bf0027bd1b/packages/extension-api/src/extension-api.d.ts#L1970

However for the `extension-api` we have something different, with an interface called `SimpleContainerInfo` which is some extending of dockerode.

https://github.com/containers/podman-desktop/blob/6c5b7848206fe65cda848be4a4d8185e19cd183b/packages/main/src/plugin/api/container-info.ts#L53

And I am wondering if we change/fix it at one place we would not have to propagate the changes also elsewhere... Waiting for @benoitf opinion on the matter